### PR TITLE
[plugin.video.sarpur] 5.0.0

### PR DESCRIPTION
**Pull request by Dagur**
### Description
Converted existing plugin to python3 and updated addon.xml and folder structure to comply with new setup rules

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.video.sarpur/addon.xml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en">Play media off the Icelandic Public Radio's website</summary>
+        <description lang="en">An Icelandic IP-Address is required to use this because of geoblocking</description>
+        <platform>all</platform>
+        <language>is</language>
+        <license>MIT</license>
+        <forum>https://spjall.vaktin.is/viewtopic.php?p=538688</forum>
+        <website>https://github.com/Dagur/sarpur-xbmc</website>
+        <source>https://github.com/Dagur/sarpur-xbmc</source>
+        <email></email>
```
Comment (enen92) [inline plugin.video.sarpur/addon.xml]: drop if empty

File: plugin.video.sarpur/addon.xml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        <import addon="script.module.beautifulsoup4" version="4.3.2"/>
+        <import addon="script.module.simplejson" version="2.0.10"/>
+        <import addon="script.module.requests" version="2.3.0"/>
+    </requires>
+    <extension point="xbmc.python.pluginsource"
+               library="default.py">
+        <provides>audio video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en">Play media off the Icelandic Public Radio's website</summary>
```
Comment (enen92) [inline plugin.video.sarpur/addon.xml]: en_GB

File: plugin.video.sarpur/addon.xml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        <description lang="en">An Icelandic IP-Address is required to use this because of geoblocking</description>
```
Comment (enen92) [inline plugin.video.sarpur/addon.xml]: en_GB

File: plugin.video.sarpur/addon.xml [hunk 4]
```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.sarpur"
+       name="Sarpur"
+       version="5.0"
+       provider-name="Dagur Páll Ammendrup">
+    <requires>
+        <import addon="xbmc.python" version="3.0.0"/>
+        <import addon="script.module.beautifulsoup4" version="4.3.2"/>
+        <import addon="script.module.simplejson" version="2.0.10"/>
```
Comment (enen92) [inline plugin.video.sarpur/addon.xml]: please remove simplejson dependency, no longer needed. Use json only.

File: plugin.video.sarpur/addon.xml [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+        <license>MIT</license>
+        <forum>https://spjall.vaktin.is/viewtopic.php?p=538688</forum>
+        <website>https://github.com/Dagur/sarpur-xbmc</website>
+        <source>https://github.com/Dagur/sarpur-xbmc</source>
+        <email></email>
+        <assets>
+            <icon>resources/icon.png</icon>
+            <fanart>resources/fanart.jpg</fanart>
+        </assets>
+        <news>Version 5.0 (2021-02-23)
```
Comment (enen92) [inline plugin.video.sarpur/addon.xml:28]: please keep this only for the latest version

**Review by enen92 — CHANGES_REQUESTED** [review]
Minors

**Comment by Dagur** [comment]
Thanks for the review. I've updated addon.xml. Should i create a new PR or is this fine?

**Comment by enen92** [comment]
This one is fine

**Review by enen92 — APPROVED** [review]

**Pull request merged**